README.md

```diff
@@ -79,10 +79,11 @@ for inclusion in the work by you, as defined in the Apache-2.0 license, shall be
 dual licensed as above, without any additional terms or conditions.
 </sub>
 
-## Third-party middleware
+## Third-party middlewares
```

Comment:
this would break any links to this section.  I dont think it is worth it to pluralise this, as that is implicit.

Reply:
Sorry, it's been a long time. I've been busy on other things and completely forget this.
Thanks for the review, it's been fixed!